src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html

```diff
@@ -96,8 +96,6 @@ <h4 *ngIf="displaySubsection(0, 3)">Student Records</h4>
       </li>
     </ol>
   </tm-instructor-help-panel>
-  <!-- Section -->
-  <h4 *ngIf="displaySubsection(3, 5)">Finding Students</h4>
   <!--Question -->
   <tm-instructor-help-panel class="instr-help-qn-first" #question [hidden]="key && !showQuestion.includes(StudentsSectionQuestions.STUDENT_SEARCH)"
```

Comment:
One last thing, the first and last questions of each section have the class `instr-help-qn-first` and `instr-help-qn-last` respectively which applies some padding. Since we removed one section and one qns, need to ensure this class is applied correctly to all the questions in the two different sections.

Reply:
Thank you for pointing so many points I missed out. I have updated this in the last commit.